changelog

```diff
@@ -1,3 +1,193 @@
+      * Clarify the description of the Far Osk 27. (@ziproot)
+      * Remove outfit scanning capabilities from Grey's person ship. (@mOctave)
+      * Block "Lunarium: Questions" from completing while assisting the Heliarchs. (@ziproot)
+      * Reword Deep's description. (@roadrunner56)
+      * List correct attributions for Unsplash images. (@ziproot)
+      * Add back instances of "ser" and explain what it means in the Skadenga and Stones of Our Fathers missions. (@bene_dictator)
+      * The Terraforming missions are no longer offered at non-sensical times. (@lumbar527)
+      * Fixed an inconsistency in condition names used by Patir Mystery and Cognizance. (@OcelotWalrus)
+      * Don't select Graffias as a target for Free Worlds "Scouting Run" jobs. (@warp-core)
+      * "Deep: Remnant: Continue Research" no longer offers if you've completed the missions after it. (@Saugia)
+      * Adjusted the explosion stats for the Korsmanath A'awoj and Echo-galleon. (@Quantumshark)
+    * Engine bugs:
+      * Fix edge case in code style checker. (@tibetiroka)
+      * Don't consider "Parsing: <file>" output when in debug mode as errors when test parsing. (@warp-core)
+      * Fix game crash on short error messages. (@tibetiroka)
+      * Don't draw name tooltips for ships hidden under the shop sidebar footer. (@TomGoodIdea)
+      * Variant ships correctly inherit display names from base ships. (@TomGoodIdea)
+      * Use singular "ship" in messages when recalling/deploying a single ship. (@tibetiroka)
+      * Fix errors in the copyright file. (@tibetiroka)
+      * Rename variable to avoid ambiguity with std::queue in some versions of clang. (@quyykk)
+      * Fix ship parenting on takeoff with fighter flagship. (@tibetiroka)
+      * Correctly handle arrow keys with no item selected in the shop. (@TomGoodIdea)
+      * Align plugin checkbox clickzones correctly. (@warp-core)
+      * Fix mission distance calculations with different settings. (@warp-core)
+      * Improvements to fix crashes when loading/unloading images. (@quyykk)
+      * Fix custom port button name when using "add port". (@warp-core)
+      * Corrected the parsing of the initial player repuptation grandchild node in Government. (@xxxyyyqqq12345)
+      * Draw distance to the selected system after mission markers. (@TomGoodIdea)
+      * Ships with "disables" personality ask for help disabling, instead of destroying, their enemies. (@TomGoodIdea)
+      * Suppress missions require landing in the next system messages from failed missions. (@OcelotWalrus)
+      * Remove outdated watchdog. (@petervdmeer)
+      * Properly check if a mission is failed. (@TomGoodIdea)
+      * Hardpoint labels and lines in the ship info panel will properly align to the "weapons" box from the interface definition. (@Zitchas)
+  * Game content:
+    * New content:
+      * Add a mission transporting a Hai to Unfettered space. (@roadrunner56)
+      * add Unfettered-related Hai news. (@Hurleveur)
```

Comment:
```suggestion
      * Add Unfettered-related Hai news. (@Hurleveur)
```

Reply:
```suggestion
      * Added Unfettered-related Hai news. (@Hurleveur)
```

Reply:
Should be `Added`; included in my comment on lines 52 - 56